pkg/gcp/gcp.go

```diff
@@ -33,14 +33,11 @@ const (
 	messageRetrievedZones             = "Retrieved the zones"
 	messageValidateCurrentGWCount     = "Verifying if current gateways match the required number of gateways"
 	messageValidatedCurrentGWs        = "Current gateways match the required number of gateways"
-	messageDeployGatewayNode          = "Deploying gateway node"
 	messageDeployedGatewayNode        = "Successfully deployed gateway node"
 	messageInsufficientZonesForDeploy = "there are insufficient zone instances to deploy the required number of gateways"
 	messageCreateExtFWRules           = "Configuring the required firewall rules for inter-cluster traffic"
-	messageDeleteExtFWRules           = "Deleting the Submariner gateway firewall rules"
+	messageDeleteExtFWRules           = "Retrieving the Submariner gateway firewall rules"
```

Comment:
Why was this changed? Now the text doesn't match the constant name.

Reply:
Sure, I'll rename the constant accordingly. Basically, I wanted the output to be as follows, so changed the text.

```
[sgaddam@localhost submariner-operator]$ ./bin/subctl cloud cleanup gcp --ocp-metadata ~/workspace-intellij/aws-ocp/sgaddam-
gcp-cp/metadata.json 
 ✓ Retrieving GCP credentials from your GCP configuration
 ✓ Initializing GCP connectivity
 ✓ Retrieving the Submariner gateway firewall rules
 ✓ Deleting firewall rule "sgaddam-gcp-cp-wrskd-submariner-public-ports-ingress" on GCP 
 ✓ Deleted firewall rule "sgaddam-gcp-cp-wrskd-submariner-public-ports-ingress" on GCP
 ✓ Successfully deleted the firewall rules
 ✓ Retrieving zones in the project 
...
```